app/controllers/users_controller.rb

```diff
@@ -16,8 +16,11 @@ def new
   end
 
   def edit
-    @people = Person.pluck(:id, [:name, :phone, :email])
-    set_form_dropdowns
+    if @user == current_user
+      redirect_to edit_user_registration_path
```

Comment:
using built-in devise route here, but admins prob need to manually set someone's password?